src/components/Header/styled.js

```diff
@@ -10,25 +10,24 @@ export const Container = styled.div`
     flex-direction: column;
     align-items: center;
   }
+`;
 
-  img {
-    cursor: pointer;
-  }
-
-  .shopping-cart {
-    width: 50px;
-  }
+export const Img = styled.img`
+  cursor: pointer;
+`;
 
-  input {
-    width: 50%;
-    box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2);
-    padding: 14px 20px;
-    margin: 8px 0;
-    border: none;
-    border-radius: 4px;
+export const ShoppingCart = styled.img`
```

Comment:
`ShoppingCart` doesn't seem appropiate for this. Perhaps `Icon` would be better. If you reuse this styled component again, most likely it won't have the same image, hence `ShoppingCart` would be a misleading name for this new image.